satpy/etc/readers/seviri_l2_bufr.yaml

```diff
@@ -3,7 +3,7 @@ reader:
   short_name: SEVIRI l2 BUFR
   long_name: MSG (Meteosat 8 to 11) Level 2 products in BUFR format
   description: SEVIRI L2 BUFR Product Reader
-  status: Alpha
```

Comment:
status Alpha is correct, revert this

Reply:
Removed "AMV BUFR products not supported yet"